### ISSUE TITLE
Is "accessible-text-block" a part of accessibility strategy?

### ISSUE THREAD

**Issue by chucklorenz** [adaptlearning/adapt_framework#833]
This snippet from line 272 of [jquery.a11y.js](https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/libraries/jquery.a11y.js):  

``` javascript
//MAKES AN ELEMENT TABBABLE  
 function makeElementTabbable($element) {  
      $element.attr({
              "role": "region",
               "tabindex": 0,
       }).addClass("prevent-default").addClass("accessible-text-block");
       return $element;
 }
```

It adds class "accessible-text-block" yet I can find no other code or style that references "accessible-text-block". Narrative, Hot Graphic, PageLevelProgress, Box Menu, and Tabs include it in their hbs templates. But it does not appear in their Less (neither in Vanilla). Using Box Menu as a model, I just included it in a PR for Cover Menu.

Is "accessible-text-block" still a part of Adapt's accessibility strategy or can the references in these plug-ins be deleted?

**Comment by oliverfoster** [comment]
It was originally going to be for colour contrast. But it seems that can be better achieved with appropriate theming. What do you think?

**Comment by chucklorenz** [comment]
I'm very pro-accessibility, but admittedly new to its implementation. I'm afraid my opinion shouldn't carry much weight.

I have wondered what a series of components might look like if they were conceived and constructed for use by the low-vision community primarily and secondarily by those with healthy vision. If this function/class might lend itself to producing such and at the same time not burden the framework, I'd want to leave it in. 

At the same time, if there is no style currently associated with it, I'd be inclined to remove the class from the hbs templates. Little pieces of inconsistently applied code like this can eat up a lot of time when some one like me is clearing tickets. I just can't leave it be! :smile:

**Comment by chris-steele** [comment]
Just to weigh in on this: given the elapsed time and the fact nothing has been done to use this I would say let's remove the references to this. Nothing is lost by doing this. @chucklorenz, @oliverfoster you both OK with this?

**Comment by oliverfoster** [comment]
sure

**Comment by moloko** [comment]
is this done?

**Comment by chris-steele** [comment]
Still present in jquery.a11y.js. There are also a handful of core plugins that use the class `accessible-text-block`. Everyone still OK for these references to be removed?

**Comment by moloko** [comment]
any update on this ?

**Comment by chucklorenz** [comment]
Yes, I'm fine with this being eliminated (but I'm still ignorant of the original vision). But it seems that issues should be created for updating the core plug-ins that still reference it, right?

**Comment by oliverfoster** [comment]
I put it in to allow us to do high-contrast text. this is the only class that gets added for a11y-text containers. I didn't ever get around to it because there was no demand.

**Comment by tomgreenfield** [comment]
- [ ] Box Menu
- [ ] Hot Graphic
- [ ] jquery.a11y
- [ ] Narrative
- [ ] Page Level Progress

**Comment by oliverfoster** [comment]
So, just PRs needed now?
